src/test/java/org/interledger/psk/AesEncryptionTest.java

```diff
@@ -0,0 +1,57 @@
+package org.interledger.psk;
+
+import org.junit.Test;
+
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Arrays;
+
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.KeyGenerator;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.spec.GCMParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+
+/**
+ * Test that locally installed provider is working as expected.
+ * 
+ * <p>To use 256-bit AES keys you must have Java Cryptography Extension (JCE) Unlimited Strength
+ * Jurisdiction Policy Files installed.
+ * 
+ * <p>{@link http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html}
+ */
+public class AesEncryptionTest {
```

Comment:
maybe im missing something, but is this test **just** to act as a canary for having JCE installed? I guess its a nice convenience for people setting up, since getting NoSuchAlgorithm can be misleading.
Perhaps lets rename the test to be more meaningful, like Aes256KeyLengthTest, and maybe have an explicit check
try {
... do something that requires AES-256
} catch(NoSuchAlgorithmException nsa) {
fail("AES-256 key could not be used, please check JCE policy is installed, nsa);
}